index.html

```diff
@@ -69,7 +69,8 @@
             </div>
             <div class="links">
                     <a id="artlink" class="artlink" target="_blank" href="https://www.space.com/2196-spacex-inaugural-falcon-1-rocket-lost-launch.html">
-                        <button class="btn youBtn">View Full Article</button>
```

Comment:
You must be using some code formatting tool and hence even for a minor `CSS btn` adjustment, multiple lines have been modified. For future `PRs`, you can try and avoid unnecessary code changes, otherwise, the `PR` is great. :)